tsconfig-strict.json

```diff
@@ -834,20 +770,28 @@
     "core/templates/domain/promo_bar/*.ts",
     "core/templates/domain/recommendations/*.ts",
     "core/templates/domain/review_test/*.ts",
+    "core/templates/domain/state/*.ts",
     "core/templates/domain/state_card/*.ts",
     "core/templates/domain/story_viewer/*.ts",
     "core/templates/domain/suggestion/*.ts",
+    "core/templates/domain/summary/*.ts",
     "core/templates/domain/topics_and_skills_dashboard/*.ts",
     "core/templates/domain/user/*.ts",
     "core/templates/domain/utilities/*.ts",
+    "core/templates/expressions/*.ts",
     "core/templates/i18n/*.ts",
     "core/templates/pages/delete-account-page/services/*.ts",
     "core/templates/pages/preferences-page/form-fields/*.ts",
     "core/templates/pages/preferences-page/modal-templates/*.ts",
+    "core/templates/pages/topics-and-skills-dashboard-page/navbar/*.ts",
     "core/templates/services/contextual/*.ts",
     "core/templates/services/stateful/*.ts",
-    "extensions/rich_text_components/Link/*.ts",
```

Comment:
Why is this removed ? `"extensions/rich_text_components/Link/*.ts",` and `"extensions/rich_text_components/Link/directives/*.ts",` both should be present ditto for others.

Reply:
ah yes, updated